cmd/Makefile.am

```diff
@@ -465,6 +465,14 @@ libexec_PROGRAMS += snapd-generator/snapd-generator
 snapd_generator_snapd_generator_SOURCES = snapd-generator/main.c
 snapd_generator_snapd_generator_LDADD = libsnap-confine-private.a
 
+##
+## snapd-env-generator
+##
+
+libexec_PROGRAMS += snapd-env-generator/snapd-env-generator
```

Comment:
Just for the record (before I forget about that again), ideally we should have something like:
```
systemdsystemenvgeneratordir=$(SYSTEMD_SYSTEM_ENV_GENERATOR_DIR)
systemdsystemenvgenerator_PROGRAMS = snapd-env-generator/snapd-env-generator
```
and drop the unnecessary `mv ../snapd/snapd-generator /usr/lib/systemd..`  in packaging.

As a side note, `pkg-config --print-variables systemd` does not list any variable that points to system env generator directory :-1: 

Obviously no reason to fix it in this PR.